src/windshaft/error.js

```diff
@@ -14,7 +14,7 @@ var WindshaftError = function (error) {
   }
   if (this.isAnalysisError(error.type)) {
     this.context = error.analysis && error.analysis.context;
-    this.analysisId = error.analysis && error.analysis.node_id;
+    this.analysisId = error.analysis && (error.analysis.node_id || error.analysis.id);
```

Comment:
Because `error.analysis.node_id` is not populated but `error.analysis.id` is. And it points to the right node in this particular node.